tests/devices/test_tetramm.py

```diff
@@ -266,11 +266,6 @@ async def test_pilatus_controller(
 
 
 async def assert_armed(driver: TetrammDriver) -> None:
-    sample_time = await driver.sample_time.get_value()
-    samples_per_reading = int(VALID_TEST_EXPOSURE_TIME / sample_time)
-    exposure = samples_per_reading * sample_time
-
     assert (await driver.trigger_mode.get_value()) is TetrammTrigger.EXT_TRIGGER
-    assert (await driver.averaging_time.get_value()) == exposure
```

Comment:
Should: Probably still assert that the averaging time is correct, both in the case of 0 exposures and >0 exposures